apps/api-docs-app/src/docs/serviceDocs.ts

```diff
@@ -111,7 +109,8 @@ class ServiceDocsImpl {
     const namespaces = this.cache.keys();
     let mergedDocs: Record<string, ServiceDoc> = {};
     const docs: Record<string, ServiceDoc> = {};
-
+    console.log('id', id);
```

Comment:
do we need these console.logs?

Reply:
looks good except for the console logs that i am wondering about.

Reply:
Thanks, I remove those console.log.